### PR TITLE
(fix) 03-1857: Make the disabled condition a label instead of a disabled search.

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary
- Display the condition as a label in edit mode.

## Screenshots

https://user-images.githubusercontent.com/30952856/218719218-ccec042e-aa1c-4373-a63a-b70ee162acac.mov

## Related Issue
- https://issues.openmrs.org/browse/O3-1857

### CODE REVIEW

**Comment by hadijahkyampeire** [comment]
@denniskigen I have implemented the feedback you shared during the coffee break.

File: packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx [hunk 1]
```diff
@@ -222,59 +223,71 @@ const ConditionsWidget: React.FC<ConditionsWidgetProps> = ({
...
-                      <span>
-                        {t('noResultsFor', 'No results for')} <strong>"{conditionToLookup}"</strong>
-                      </span>
-                    </Tile>
-                  </Layer>
-                </>
-              );
-            })()}
-          </div>
+          {context === 'editing' ? (
```
Comment (denniskigen) [inline packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx:226]: When editing a condition, we should show the `display` value property from the `condition` prop. This value is immediately available, unlike the `selectedCondition` property. That means we shouldn't need to show a loading state either. This should be refactored to:

```tsx
const displayName = getFieldValue(condition?.cells, 'display');
```

Which could then get used as follows:

```tsx
{context === 'editing' ? (
  <FormLabel className={styles.conditionLabel}>. 
    {displayName}
  </FormLabel>
...
}
```

File: packages/esm-patient-conditions-app/translations/en.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
   "enterCondition": "Enter condition",
   "errorDeletingCondition": "Error deleting a condition",
   "inactive": "Inactive",
+  "loadingCondition": "Loading Condition",
```
Comment (denniskigen) [inline packages/esm-patient-conditions-app/translations/en.json:27]: This translation shouldn't be required based on the advice above.

**Review by denniskigen — COMMENTED** [review]
This is very close @hadijahkyampeire. Just a few issues to address.

File: packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx (hunk 1 above)
Comment (hadijahkyampeire) [reply packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx:226]: oh okay, I see that, thanks.

**Comment by hadijahkyampeire** [comment]
@denniskigen I have made the tweaks and done some small refactoring since you have work for the most refactoring.

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks, @hadijahkyampeire!

**Pull request merged**